tensorboard/webapp/runs/data_source/runs_backend_types.ts

```diff
@@ -101,10 +102,12 @@ export interface BackendHparamsExperimentResponse {
 
 interface HparamsColFilterParams {
   hparam: string;
+  includeInResult: boolean;
```

Comment:
Can you explain what this is doing? What is a result? Why would something not be included?

I'm kinda assuming this is a flag returned by the server which indicates that an hparam was included in the response, but that seems kinda redundant?

Reply:
api.proto probably has the best documentation for this:
https://github.com/tensorflow/tensorboard/blob/6f5be4bca32f78956d83ca1d4caed622f0f35a3d/tensorboard/plugins/hparams/api.proto#L385

I'm following the pattern of the rest of the file of not repeating the documentation here.

The api.proto documentation is not perfect, however. Left unsaid is that if you don't set includeInResult field on any of the fields in the request then the backend assumes you want all fields, even if they are not specified as a col param. I believe this is for backward compatibility.

https://github.com/tensorflow/tensorboard/blob/6f5be4bca32f78956d83ca1d4caed622f0f35a3d/tensorboard/plugins/hparams/list_session_groups.py#L988

So we have to set includeInResult or the backend would return hparam values for all hparams and not just the 1000 we retrieved in the /experiment request.

Reply:
Thanks for the link, I think I understand now.